src/platforms/atomic-kms/server/gbm_display_allocator.cpp

```diff
@@ -151,9 +151,21 @@ auto create_gbm_surface(
                     modifiers.size(),
                     flags);
             }
-        }();
+        };
+
+    // Try allocating with flags first
+    // Covers Nvidia cards with 575 drivers and later
+    auto surface = try_create_surface(GBM_BO_USE_RENDERING | GBM_BO_USE_SCANOUT);
 
     if (!surface)
+    {
+        mir::log_warning("Failed to create surface with flags. Attempting to create one without flags");
```

Comment:
I don't think this is something someone running Mir needs care about, so a "warning" seems excessive.

```suggestion
        mir::log_info("Failed to create display buffer surface with correct flags. Attempting to create one without flags");
```